uavcan/driver/__init__.py

```diff
@@ -11,13 +11,19 @@
 import sys
 from .slcan import SLCAN
 from .common import DriverError, CANFrame
-from .python_can import PythonCan
 
 if sys.platform.startswith('linux'):
     from .socketcan import SocketCAN
 else:
     SocketCAN = None
 
+try:
+    import can
```

Comment:
Why can't we move `from .python_can import PythonCAN` here?

Reply:
IMO it reads more explicit that we're just checking that the specific module, `can`, is present; also that way we also prevent a possible import error in `python_can.py` from being caught and ignored.

Reply:
IDK it seems non-obvious and either I'm reading too much into it but it also looks like a leaky abstraction where you make assumptions about the underlying structure of the imported module. The general lack of a well-constructed architecture here might make my observation about the leaky abstraction here a bit inappropriate but it's still valid.

If you are certain about this then I won't insist.

Reply:
I've made the change as per your request.

Reply:
That's not quite what I meant. The `import can` statement is the one I am concerned about. Any chance to get rid of it?

Reply:
Sorry, I misunderstood what you meant. I've moved the check to the `.python_can` module itself. You're right, it's cleaner that way.